demos/opencv/snap/snapcraft.yaml

```diff
@@ -1,16 +0,0 @@
-name: opencv-example
-version: 1.0
-summary: Use OpenCV and OpenGL
-description: A simple OpenCV example
-confinement: strict
-
-build-packages: [gcc, g++, libc6-dev, libopencv-dev]
```

Comment:
This was a good demo to ensure that library crawling continued working correctly. Does one of the "hello" snaps cover that case?